### DISCUSSION TITLE
bridge_v2_version field for extensions

### DISCUSSION THREAD

**Discussion by ihategravel2** [bridge-core/editor#238] [Ideas] [answered]
I'm not sure if this is the right place for this (github dumb) I think something to consider adding is a "bridge_v2_version" field to extension manifest. Anyone have any thoughts on possible use cases for this? 
```json
{
	"author": "Extension Author",
	"name": "Extension Name",
	"version": "1.0.0",
	"id": "",
	"description": "Extension Description",
	"api_version": 2,
	"target": "v2",
        "bridge_v2_version": [">=", "2.1.5"], //This
	"tags": [
		"Presets"
	]
}
```

**Comment by solvedDev** [comment]
Yeah, I've been thinking of re-adding something like the "min_app_version" we had on v1. I would like to support setting minimum and maximum values though:

```json
{
  "supportsAppVersion": {
    "min": "2.1.0",
    "max": "2.2.0"
  }
}
```

**Comment by solvedDev** [comment, marked as answer]
Corresponding issue: #241
